game/script.rpy

```diff
@@ -922,10 +925,212 @@ label conservatory_menu:
 
     menu:
 
+        "I look in the drawer":
+
+            jump pasiphae_play_intro
+
         "I move on":
 
             jump interlude
 
+label pasiphae_play_intro:
+
+  "In the drawer I found a book with neatly penned pages. This penmanship was neat, evenly spaced, and deliberately cursive. It seemed this book may not have been meant for publishing. The lack of title, and author's name helped confirm my suspicions."
+
+  menu:
+
+    "This must be one of James' unpublished works":
+      $ pasiphae_author = "James"
+      jump pasiphae_play
+
+    "This must have been written by someone other than James":
+      jump pasiphae_play_author
+
+label pasiphae_play_author:
+
+  "But who else could it have been?"
+
+  menu:
+
+    "A guest must have left this here and forgotten about it":
+      jump pasiphae_play_author_mystery
+
+    "Someone must have spent a lot of time here writing it":
+      jump pasiphae_play_author_marian
+
+label pasiphae_play_author_mystery:
+  $ pasiphae_author = "this unknown author"
+
+  "The author of this book must have thought very little of his or her work to leave it here."
+
+  "Probably some very drunk or very happy party guest."
+
+  jump pasiphae_play
+
+label pasiphae_play_author_marian:
+  $ pasiphae_author = "Marian"
+
+  "The only other person who lived here was Marian. This must have been from her hand."
+
+  jump pasiphae_play
+
+label pasiphae_play:
+
+if tutor:
```

Comment:
I like the idea of the tutor recognizing the characters. 
I like both scenes, but I think that there should be one. This would not have changed based on the protagonist's occupation